src/megameklab/com/util/UnitPrintManager.java

```diff
@@ -258,9 +255,12 @@ private static File getExportFile(Frame parent, String suggestedFileName) {
         return sheets;
     }
 
-    public static boolean exportUnits(List<Entity> units, File exportFile, boolean singlePrint) {
-        List<PrintRecordSheet> sheets = createSheets(units, singlePrint);
+    public static void exportUnits(List<Entity> units, File exportFile, boolean singlePrint) {
+        RecordSheetOptions options = new RecordSheetOptions();
+        List<PrintRecordSheet> sheets = createSheets(units, singlePrint, options);
         PageFormat pageFormat = new PageFormat();
+        // getPaper() returns a clone
+        pageFormat.setPaper(options.getPaperSize().createPaper());
```

Comment:
This seems off to me.

Reply:
That comment is obsolete. Is it the disconnect between the comment and the code that's off?

Reply:
It is